src/lib/syncPagePath.ts

```diff
@@ -208,6 +208,23 @@ export default async function syncPagePath(this: CommandsThatSyncMarkdown) {
     throw err;
   });
 
+  // check whether or not the project has bidirection syncing enabled
+  // if it is, validations must be skipped to prevent frontmatter from being overwritten
+  const headers = new Headers({ authorization: `Bearer ${key}` });
+  const projectMetadata: ProjectRepresentation = await this.readmeAPIFetch('/projects/me', {
+    method: 'GET',
+    headers,
+  }).then(res => {
+    return this.handleAPIRes(res);
+  });
+
+  const biDiConnection = projectMetadata.data.git?.connection;
```

Comment:
maybe just to be extra cautious:

```suggestion
  const biDiConnection = projectMetadata?.data?.git?.connection;
```